Formula/prs.rb

```diff
@@ -0,0 +1,33 @@
+class Prs < Formula
+  desc "Secure, fast & convenient password manager CLI with GPG & git sync"
+  homepage "https://timvisee.com/projects/prs"
+  url "https://github.com/timvisee/prs/archive/v0.4.1.tar.gz"
+  sha256 "f7f8f5d815cf1c4034f1c2aa36ed29b7e3ab2a884791b4b69ffc845d5d111524"
+  license "GPL-3.0-only"
+
+  depends_on "rust" => :build
+  depends_on "gpgme"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "libxcb"
+    depends_on "openssl@3"
+  end
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "./cli")
+
+    system bin/"prs", "internal", "completions", "--output=completions/", "bash", "fish", "zsh"
+    bash_completion.install "completions/prs.bash"
+    fish_completion.install "completions/prs.fish"
+    zsh_completion.install "completions/_prs"
```

Comment:
```suggestion
    system "cargo", "install", *std_cargo_args(path: "cli")

    generate_completions_from_executable(bin/"prs", "internal", "completions")
```

Reply:
Good suggestion, though I wonder if it works this way.

This is what I could find as documentation on the function: https://rubydoc.brew.sh/Formula#generate_completions_from_executable-instance_method

Does `generate_completions_from_executable` expect the completion scripts from stdout or from a file? Because currently the completion scripts are written to a file, unless `--stdout` is given.

Reply:
yeah, I already tested, it works fine.

Reply:
You can test yourself

Reply:
Fantastic! I'll add this and force push.